CHANGELOG.md

```diff
@@ -18,7 +18,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Now has concise contribution guidelines
 - Use Furo theme for documentation
 
-
```

Comment:
Here, I made a dummy changelog modification to get through our pipeline checks. I think we should make that check optional, it should only serve as a reminder for us, but is not always needed (as in this case). @Scienfitz: if you agree, could you perhaps replace this commit with the corresponding change in our pipeline?

Reply:
As written in the other post: Since Martin has no strong opinion, we can go with your approach.

Reply:
optional pipeline step sounds useful, how would I trigger the skipping of the changelog step? thats then manual somewhere right?

Reply:
should for instance not be possible for random contributors

Reply:
Yes, so my desired version would be that the changelog still shows up "red" on the checks but you can merge nevertheless (there is zero chance that one could overlook this, I think)

And I hope that we have set up the repo in a way that only CODEOWNERS can merge?

Reply:
@Scienfitz: so I guess I'll wait until you added the optional step for the changelog (in fact, it seems it is required already for this PR)

Still, could you confirm the CODEOWNERS setting?

Reply:
resolving as i will make the changelog optional in another PR

Reply:
@AdrianSosic  not aware of any such setting, only code owners setting is require them as reviewer which is fine as they will see the missing readme too)